tests/lapax_test.py

```diff
@@ -30,6 +30,10 @@
 from jax.config import config
 config.parse_flags_with_absl()
 
+# Primitive 
+
+_cpu_lapack_types = [onp.float32, onp.float64]
```

Comment:
This list of types actually has nothing to do with CPU or LAPACK. This module tests `jax.experimental.lapax`, which is an experimental/toy reimplementation of a few `lapack` primitives on top of `jax`.

But if you would instead name it something like `float_types` instead, I'd agree with the change, since it's better to have a single global list of types to test rather than two copies of it.

Reply:
@hawkinsp yup made the changes